test/sdn_migration/scripts/validate-sdn-ovn-migration-jobs.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set +e
+
+UNIQUE_ID=$(date +%s%N) # Generate a unique identifier for this instance
+CONTAINER_NAME="sdn-ovn-migration-${UNIQUE_ID}" # Name of the container based on the unique identifier
+
+# Check if the container already exists
+if podman ps -a --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
+    echo "Container ${CONTAINER_NAME} already exists. Exiting."
+    exit 1
+fi
+
+# Create the container with environment variables and unique name
+podman create --name "${CONTAINER_NAME}" -e OCM_TOKEN \
```

Comment:
cc @MrSantamaria @ritmun (it seems this was copied from previous scripts https://github.com/openshift/osde2e/blob/main/scripts/gap-analysis-jobs.sh )

Why do we create the container here, then attach to it immediately instead of just `podman run ...` with a volume mount of the current directory? 4 commands can be done in 1 :smile: